cubedash/_monitoring.py

```diff
@@ -23,7 +24,7 @@ def init_app_monitoring():
 
     _INITIALISED = True
 
-    @_model.app.before_request
+    @current_app.before_request
```

Comment:
I don't think this will work if monitoring is enabled, current_app won't be anything real when this is called.

I think instead, app should be passed as an argument to the `init_app_monitoring()`